src/controllers/controllerengine.cpp

```diff
@@ -1440,6 +1440,23 @@ void ControllerEngine::softTakeover(QString group, QString name, bool set) {
 }
 
 /*  -------- ------------------------------------------------------
+     Purpose: Ignores the next value for the given ControlObject
+                This is used when an absolute physical control is changed to
+                to operate on a different ControlObject, allowing it to sync up
+                to the soft-takeover state
+     Input:   ControlObject group and key values
+     Output:  -
+     -------- ------------------------------------------------------ */
+void ControllerEngine::softTakeoverResync(QString group, QString name) {
```

Comment:
I'd just prefer everything to be called one thing -- maybe rename ignoreNext on the SoftTakeover side then ?